main/lsp/requests/completion.cc

```diff
@@ -1115,6 +1115,11 @@ unique_ptr<ResponseMessage> CompletionTask::runRequest(LSPTypecheckerInterface &
     auto resp = move(queryResponses[0]);
 
     if (auto sendResp = resp->isSend()) {
+        if (sendResp->dispatchResult->main.receiver.isUntyped()) {
+            response->result = std::move(emptyResult);
+            return response;
+        }
```

Comment:
Can we push this into similarMethodsForClass?

Reply:
I moved this into `allSimilarMethods`, which seemed like a better choice.